cmake/modules/AddSwiftUnittests.cmake

```diff
@@ -89,7 +111,8 @@ function(add_swift_unittest test_dirname)
     endif()
   endif()
 
-  if (SWIFT_SWIFT_PARSER)
+  if (SWIFT_SWIFT_PARSER AND NOT ASU_IS_TARGET_TEST)
+    # Link to stdlib the compiler uses.
```

Comment:
e.g. `runtime` tests is a "target" test, not "host" test. "target" test should not link with host libraries.